tests/cassettes/TestReferralCustomerAddCreditCard.yaml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Delete this cassette since it won't be used

Reply:
Resolved